framework/db/mssql/Schema.php

```diff
@@ -189,6 +189,7 @@ protected function loadColumnSchema($info)
         $column->comment = $info['comment'] === null ? '' : $info['comment'];
 
         $column->type = self::TYPE_STRING;
+        $column->size = $info['character_maximum_length'];
```

Comment:
According to the [MSDN article](https://msdn.microsoft.com/en-us/library/ms188348.aspx), `character_maximum_length` is maximum length, in characters, for binary data, character data, or text and image data.

Returned values:
-1 for xml and large-value type data. Otherwise, NULL is returned. For more information, see Data Types (Transact-SQL).

We need to handle `-1` properly